src/main/resources/mcmod.info

```diff
@@ -1,14 +1,14 @@
 [
 {
-  "modid": "ReAuth",
```

Comment:
Changed to lower case as server and client logs all say that modid's need to be lower case.